app/models/invoice.rb

```diff
@@ -8,4 +8,12 @@ class Invoice < ApplicationRecord
   has_many :items, through: :invoice_items
 
   validates :status, presence: true
+
+  def items_belonging_to
+    invoice_items.joins(:item).select('invoice_items.status, invoice_items.quantity, invoice_items.id as invoice_item_id, items.*')
+  end
```

Comment:
I think you could DRY this up by using just `invoice.items`

Reply:
Agreed, nice catch Scott